src/SmartTransactionsController.ts

```diff
@@ -889,4 +889,56 @@ export default class SmartTransactionsController extends StaticIntervalPollingCo
       );
     });
   }
+
+  #wipeSmartTransactionsPerChainId(
```

Comment:
Minor and may not be the case in this file, but typically we try and group public methods together at the top with private methods below them, for easier visibility.